prober/http.go

```diff
@@ -296,6 +350,11 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 			Help: "Indicates if probe failed due to regex",
 		})
 
+		probeFailedDueToCel = prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "probe_failed_due_to_cel",
+			Help: "Indicates if probe failed due to CEL",
```

Comment:
`due to CEL` is missing a qualifier, how about `due to CEL not matching`

Reply:
This works similarily to the regex one, so if the probe fails due to the CEL expression matching or CEL expression not matching it will be increased in both cases.